device/iot-device-samples/android-sample/app/src/main/res/layout/activity_main.xml

```diff
@@ -48,24 +48,4 @@
         android:onClick="btnUpdateReportedOnClick"
         android:text="Update Reported Properties" />
 
-    <Button
-        android:id="@+id/btnFileUpload"
```

Comment:
Getting rid of the file upload button since it makes this sample more complicated than it needs to be.